packages/cli-hermes/src/profileHermes/sourcemapUtils.ts

```diff
@@ -0,0 +1,120 @@
+import {Config} from '@react-native-community/cli-types';
+import {logger, CLIError} from '@react-native-community/cli-tools';
+import fs from 'fs';
+import path from 'path';
+import os from 'os';
+import {SourceMap} from 'hermes-profile-transformer';
+import ip from 'ip';
+import nodeFetch from 'node-fetch';
+
+function getTempFilePath(filename: string) {
+  return path.join(os.tmpdir(), filename);
+}
+
+function writeJsonSync(targetPath: string, data: any) {
+  let json;
+  try {
+    json = JSON.stringify(data);
+  } catch (e) {
+    logger.error(
+      `Failed to serialize data to json before writing to ${targetPath}\n`,
+      e,
+    );
+  }
+
+  try {
+    fs.writeFileSync(targetPath, json, 'utf-8');
+  } catch (e) {
+    logger.error(`Failed to write json to ${targetPath}\n`, e);
+  }
+}
+
+async function getSourcemapFromServer(): Promise<SourceMap> {
+  logger.debug('Getting source maps from Metro packager server\n');
+  const DEBUG_SERVER_PORT = '8081';
```

Comment:
port is hardcoded. Worth to pass it from arguments and default to 8081

Reply:
Yup I've added another flag `--port [number]` so user can enter their running server port (default is 8081)